python/lsst/daf/butler/core/registry.py

```diff
@@ -129,6 +130,25 @@ def setConfigRoot(cls, root, config, full, overwrite=True):
         Config.updateParameters(RegistryConfig, config, full,
                                 toCopy=(("skypix", "cls"), ("skypix", "level")), overwrite=overwrite)
 
+    @staticmethod
+    def getDialect(registryConfig):
```

Comment:
I'm not completely sold on this being a static method of Registry.  I think it would be better as a normal instance method on `RegistryConfig`.

Reply:
Moved to `RegistryConfig` as an instance method. It does work better.